help_function/src/onedpl_test_group_load.cpp

```diff
@@ -0,0 +1,152 @@
+// ====------ onedpl_test_group_load.cpp------------ *- C++ -* ----===//
+
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//
+// ===----------------------------------------------------------------------===//
+#include <sycl/sycl.hpp>
+#include <dpct/dpct.hpp>
+#include <dpct/dpl_utils.hpp>
+#include <iostream>
+#include <oneapi/dpl/iterator>
+
+bool test_load_blocked_striped(dpct::group::load_algorithm T) {

```

Comment:
`T` needs to be a template parameter.

The `using workgroup_load = dpct::group::workgroup_load<128, T, int>;` line will fail to compile since T is a runtime variable.

Instead, you could do something like:
```
template <typename LoadStrategy>
bool test_load_blocked_striped() {
    ...
    using workgroup_load = dpct::group::workgroup_load<128, LoadStrategy, int>;
    ...
}